src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -151,6 +164,10 @@ private void updateReport(final LocalDate newvalue) {
 
       this.currentDayTimeLabel.setText(DateFormatter.secondsToHHMMSS(currentSeconds));
       this.currentDayWorkTimeLabel.setText(DateFormatter.secondsToHHMMSS(currentWorkSeconds));
+
+      controller = new Controller(model, dateProvider);
+      colorTimeLine = new ColorTimeLine(reportColorTimeLine, model, controller);
+      colorTimeLine.update();
```

Comment:
what happens if I change the day? will the timeline update to this day?

Reply:
It didn't but now it does 😃